implementations/elixir/ockam/ockam/lib/ockam/wire/binary/versions.ex

```diff
@@ -78,7 +79,7 @@ defmodule Ockam.Wire.Binary.V1 do
   def decode(encoded) do
     ## Expect first byte to be the version
     case encoded do
-      <<@version, _rest::binary>> ->
+      <<@version_1, _rest::binary>> ->
```

Comment:
I didn't pick up on this at first, but in furtherance of my comment above and the general goal, I think we should pick this byte off earlier in the processing and pick the implementation as appropiate.